### PR TITLE
services: wrap transactions with fee bump for sponsored accounts

### PR DESCRIPTION
**Pull request by CaioTeixeira95**
### What

This PR adds the `POST /tx/create-fee-bump` endpoint. Also the `AccountModel` was created.

### Why

Implement the fee bump functionality.

### Known limitations

N/A

#### PR Structure

- [x] It is not possible to break this PR down into smaller PRs.
- [x] This PR does not mix refactoring changes with feature changes.
- [x] This PR's title starts with name of package that is most changed in the PR, or `all` if the changes are broad or impact many packages.

#### Thoroughness

- [x] This PR adds tests for the new functionality or fixes.
- [x] All updated queries have been tested (refer to [this](https://stackoverflow.com/a/29163465) check if the data set returned by the updated query is expected to be same as the original one).

#### Release

- [x] This is not a breaking change.
- [x] This is ready to be tested in development.
- [ ] The new functionality is gated with a feature flag if this is not ready for production.

### CODE REVIEW

File: internal/services/account_sponsorship_service.go [hunk 1]
```diff
@@ -125,10 +142,69 @@ func (s *accountSponsorshipService) SponsorAccountCreationTransaction(ctx contex
...
+			BaseFee:    int64(s.BaseFee),
+		},
+	)
+	if err != nil {
+		return "", "", fmt.Errorf("creating fee-bump transaction: %w", err)
+	}
+
+	signedFeeBumpTx, err := s.SignatureClient.SignStellarFeeBumpTransaction(ctx, feeBumpTx)
+	if err != nil {
+		return "", "", fmt.Errorf("signing fee bump transaction: %w", err)
```
Comment (daniel-burghardt) [inline internal/services/account_sponsorship_service.go:196]: mega nit: 
```suggestion
		return "", "", fmt.Errorf("signing fee-bump transaction: %w", err)
```

**Review by daniel-burghardt — APPROVED** [review]
Left some non-critical comments, but overall LGTM 🚀

File: internal/serve/serve.go [hunk 2]
```diff
@@ -125,9 +140,11 @@ func handler(deps handlerDeps) http.Handler {
 			handler := &httphandler.AccountHandler{
 				AccountSponsorshipService: deps.AccountSponsorshipService,
 				SupportedAssets:           deps.SupportedAssets,
+				BlockedOperationsTypes:    blockedOperationTypes,
```
[the remark this answers is not in the document]
Comment (CaioTeixeira95) [reply internal/serve/serve.go:143]: I liked this approach!

File: internal/services/account_sponsorship_service.go (hunk 1 above)
Comment (CaioTeixeira95) [reply internal/services/account_sponsorship_service.go:196]: lol xD

File: internal/serve/serve.go [hunk 3]
```diff
@@ -22,6 +23,20 @@ import (
 	"github.com/stellar/wallet-backend/internal/signing"
 )
 
+// NOTE: perhaps move this to a environment variable.
```
[the remark this answers is not in the document]
Comment (CaioTeixeira95) [reply internal/serve/serve.go:26]: I'll create a ticket for it!

**Review by daniel-burghardt — APPROVED** [review]

**Pull request merged**